packages/app/obojobo-express/middleware.default.js

```diff
@@ -43,10 +43,12 @@ module.exports = app => {
 			rolling: true,
 			cookie: {
 				path: '/',
-				sameSite: false, // Seems to be blocking access from webcourses2c.instructure.com to obojobo (though not webcourses.ucf.edu?)
+				maxAge: 10 * 24 * 60 * 60 * 1000, // 30 days
+				// use config.general.secureCookie to determine the next settings
+				// sameSite 'none' is needed to support LTI launches from different domains
+				sameSite: config.general.secureCookie ? 'none' : false,
```

Comment:
Do we maybe want a different config value for this instead of going with secureCookie?

Reply:
Just as a note for the future - sameSite=none requires the cookie to be secure, so they're somewhat linked together.  The only situation I could imagine wanting a separate flag for this is when you are hosting your LMS and obonext on the same domain and would like to lock down that cookie a little more.